pkg/nsx/services/staticroute/builder.go

```diff
@@ -41,31 +42,12 @@ func (service *StaticRouteService) buildStaticRoute(obj *v1alpha1.StaticRoute) (
 		nexthop.IpAddress = &obj.Spec.NextHops[index].IPAddress
 		sr.NextHops = append(sr.NextHops, nexthop)
 	}
-	sr.Id = String(fmt.Sprintf("sr_%s", fmt.Sprint(obj.UID)))
-	sr.DisplayName = String(fmt.Sprintf("sr-%s-%s", obj.Namespace, obj.Name))
+	sr.Id = String(util.GenerateID(string(obj.UID), "", "", ""))
```

Comment:
so all the other CR's id and displayname comply with this  and leave the last parameters blank?

Reply:
yes